reactDashboard/src/components/ChartDropdown/ChartDropdown.test.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import ChartDropdown from "./ChartDropdown";
+import { render, unmountComponentAtNode } from "react-dom";
+import { act } from "react-dom/test-utils";
+import {
+  DAILY_CASES,
+  TOTAL_CASES,
+  DAILY_DEATHS,
+  TOTAL_DEATHS,
+  PERCENTAGE_CASES,
+} from "../DataCharts/DataChartsConsts";
+
+var storedChartType = DAILY_CASES;
+const setChartType = (value) => (storedChartType = value);
+
+var container = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+function checkDropdownText() {
```

Comment:
Could use a couple of testcases which show menu content with and without showPercentageCases set